### ISSUE TITLE
Incompatible mod set

### ISSUE THREAD

**Issue by taddmencer** [Wurst-Imperium/WI-Zoom#15]
Just updated to 1.18.1 and decided to snag this mod for zooming since Optifine is not fine at all.

Fabric is updated to the latest.

Installed WI Zoom.

Only Reply Mod is installed.

Before I put WI Zoom in, Fabric and ReplayMod work fine.

Added this and got the Incompatible mod set error and says that it requires version 0.42.4 or later of fabric. 

Am I doing something terrible and inconsiderate here?

![image](https://user-images.githubusercontent.com/1148597/145696885-cde53c96-bc67-403a-9884-a43c610fb281.png)

**Comment by RDKRACZ** [comment]
You need to download (or update) Fabric API.

> Fabric is updated to the latest.

`fabric` is actually fabric-api. Not the Loader.

**Comment by taddmencer** [comment]
ppffttt .. ok, you know what? If all problems could be solved by simple logic, that would make sense.
And yes, it was the fabric-api I was missing. I had the API originally,but as I was updating things I think I deleted it like a boneheaded bonehead thinking I needed to do that .... cause you know, reading is hard so why read the instructions on the Fabric website?

Thanks for your help and patience, RDKRACZ. Much appreciated.
